src/LearnContent/LearnContent.tsx

```
@@ -2,14 +2,13 @@
 
-import type { LearnPost } from "../lib/learn.js";
-import { Layout } from "../wrappers/Layout.js";
-import { Date } from "../ui/Atoms/Date.js";
-
-export function LearnContent({
-  frontMatter,
-  mdxSource,
-}: LearnPost): JSX.Element {
+import type { LearnPost } from "../lib/learn";
+import { Layout } from "../wrappers/Layout";
+import { Date } from "../ui/Atoms/Date";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)